src/stack.js

```diff
@@ -1,5 +1,49 @@
-'use strict'
```

Comment:
I don't think `'use strict'` affects the spread operator.  (Maybe the internet lied to you).  This would be pretty easy to test.  If you add `'use strict'` back does `beefUpMany` still work?